manager/controllers/baseline_update.go

```diff
@@ -86,6 +88,7 @@ func BaselineUpdateHandler(c *gin.Context) {
 		LogAndRespError(c, err, "Database error")
 		return
 	}
+	kafka.EvaluateBaselineSystems(&baselineID, account)
 
 	c.JSON(http.StatusOK, baselineID)
 }
```

Comment:
I think we also need to modify tests for create/update/delete baseline to check if systems were correctly recalculated.